src/etc/man/cargo-package.1

```diff
@@ -9,7 +9,8 @@ cargo\-package \[em] Assemble the local package into a distributable tarball
 \fBcargo package\fR [\fIoptions\fR]
 .SH "DESCRIPTION"
```

Comment:
We do need to document the new flags.  They should be in the same commit as the flags